mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !/^a$/i.test(currentTarget.nodeName)) {
```

Comment:
yep, even `currentTarget.nodeName.toLowerCase() != 'a'`

I wonder if simply looking for the first parent with `currentTarget.href` wouldn't be more solid...